calico/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,49 @@
+name: calico
+files:
+  - name: calico.yaml
+    options:
+      - template: init_config
+        options:
+          - template: init_config/default
+      - template: instances
+        options:
+          - template: instances/default
+          - name: prometheus_url
+            required: true
+            description: Prometheus url and metrics to check
+            value:
+              type: string
+              example: http://localhost:9091/metrics
+          - name: metrics
```

Comment:
`metrics` already comes defined in `instances/openmetrics` although is usually hidden `extra_metrics`(which comes in the template as well) is the one usually used to collect additional metrics